.github/workflows/push-master.yml

```diff
@@ -12,7 +12,9 @@ jobs:
       - name: Checkout source code
         uses: actions/checkout@v3
         with:
-          fetch-depth: 2
+          fetch-depth: 0
+      - name: Fetch all branches and tags
+        run: git fetch --all
```

Comment:
Do not think this is needed, the fetch-depth is a left over of the old pre-split flow and we should not need it here for the code coverage we run.